test/unit/account_info/test_sqlite_account_info.py

```diff
@@ -55,3 +64,66 @@ def test_migrate_to_4(self):
                 "SELECT recommended_part_size, absolute_minimum_part_size from account"
             ).fetchone()
         assert (100, 5000000) == sizes
+
+
+class TestSqliteAccountProfileFileLocation:
+    @pytest.fixture(autouse=True)
+    def setup(self, monkeypatch):
+        monkeypatch.delenv(B2_ACCOUNT_INFO_ENV_VAR, raising=False)
+        monkeypatch.delenv(XDG_CONFIG_HOME_ENV_VAR, raising=False)
+        if os.path.exists(os.path.expanduser(B2_ACCOUNT_INFO_DEFAULT_FILE)):
+            os.remove(os.path.expanduser(B2_ACCOUNT_INFO_DEFAULT_FILE))
```

Comment:
That would delete default accountinfo on my development environment. Maybe there is a better way?